libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/MatrixClient.kt

```diff
@@ -89,8 +89,9 @@ interface MatrixClient : Closeable {
      * Returns an optional URL. When the URL is there, it should be presented to the user after logout for
      * Relying Party (RP) initiated logout on their account page.
      * @param ignoreSdkError if true, the SDK will ignore any error and delete the session data anyway.
+     * @param forced if true, the logout came from the HS, no request will be made and the data will be kept.
```

Comment:
Actually most of the data will be deleted, but we will keep the entry in the Session database, so that the application is able to display a "You've been signed out" screen ([here](https://github.com/element-hq/element-x-android/blob/develop/appnav/src/main/kotlin/io/element/android/appnav/RootFlowNode.kt#L114)). Maybe make the comment a bit clearer?